clear-site-data/support/server.py

```diff
@@ -0,0 +1,11 @@
+import json
+
+KNOWN_DATATYPES = ["cookies", "storage", "cache"]
+
+# A support server that receives a list of datatypes in the GET query
+# and returns a Clear-Site-Data header with those datatypes.
+def main(request, response):
+    types = [key for key in request.GET.keys() if key in KNOWN_DATATYPES]
+    header = json.dumps({ "types": types })
+    response = "Clear-Site-Data: %s" % header
+    return [("Clear-Site-Data", header)], response
```

Comment:
Did you consider having this page `postMessage` the state of various storage types back up to the parent? That might be simpler (and would allow you to do cross-origin tests in the future).